.github/workflows/main.yml

```
@@ -2,11 +2,13 @@
 
 name: Continuous Integration Laravel
 
-# Controls when the action will run. 
+# Controls when the action will run.
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
-    branches: [ master ]
+    branches:
+      - master
+      - bugfix_orders_tests
```

```suggestion
      - bugfix/orders_tests
```
и бранчи с багфиксом должны быть как-бы в подпапке `bugfix/`
фичи в `feature/`